src/Datepicker.js

```diff
@@ -18,10 +20,12 @@ export default class Datepicker extends Component {
     this.handleCalendarVisibility = this.handleCalendarVisibility.bind(this)
     this.dateChange = this.dateChange.bind(this)
     this.outsideHandler = null
+    this.handleChange = this.handleChange.bind(this)
 
     this.state = {
       showCalendar: false,
       selectedDate: this.props.startDate,
+      textInput: '',
```

Comment:
Shouldn't this take in the default `value` prop instead?

Reply:
Again, leaving it blank initially is consistent with current Prod datepicker functionality. But I agree that what you suggests makes more sense.